README.md

```diff
@@ -3,3 +3,41 @@
 This project is part of ASP.NET Core. You can find samples, documentation and getting started instructions for ASP.NET Core at https://github.com/aspnet/AspNetCore.
 
 Use https://github.com/aspnet/AspNetCore/issues for issues with this project.
+
+### Build on Windows ###
```

Comment:
We should mention dependencies that need to be installed on non-macOs platform like cmake for Windows and cmake/"build-essentials"/gcc/g++ on linux.

Reply:
I'll try to gather that info and improve the readme at a later time.